api/views/__init__.py

```diff
@@ -146,6 +151,18 @@ class ProfileViewSet(viewsets.ModelViewSet):
     def get_queryset(self):
         return Profile.objects.filter(user=self.request.user)
 
+    @decorators.action(detail=True, methods=["post"])
+    def report_webcompat_issue(self, request, pk=None):
+        serializer = WebcompatIssueSerializer(data=request.data)
+        if serializer.is_valid():
+            info_logger.info("webcompat_issue", extra=serializer.data)
+            incr_if_enabled(f"webcompat_issue", 1)
+            for k, v in serializer.data.items():
+                if v and k != "issue_on_domain":
+                    incr_if_enabled(f"webcompat_issue_{k}", 1)
+            return response.Response(status=status.HTTP_201_CREATED)
+        return response.Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
```

Comment:
change (blocking): Since this isn't associated with a profile, we should move it out from "under" the `ProfileViewSet` and into its own `@decorators.api_view` function.

Reply:
Making it it's own endpoint but I wanted it to be under profile in case we want to log from whom this request is coming from (to later get back to the user and/or the email address being used is specifically not being accepted because of how the user was using the alias).

Reply:
This code doesn't log the profile ID though? And having this in the `ProfileViewSet` means the permission class includes `IsOwner` which ... I'm not sure how that will work here?

Reply:
I put a debugger on the `IsOwner` permission and found out that the permission does not get checked because the Profile object is never called.

Reply:
Moved the endpoint to its own since the profile value is not being used: [847d735](https://github.com/mozilla/fx-private-relay/pull/2378/commits/847d7352df89960942705bdbe344eb2dd00566f2)